### PR TITLE
Update to absolute file paths

### PR DESCRIPTION
**Pull request by Chem-Star**
Updated the path to certain file operations to use absolute file paths defined in appsettings. This allows greater flexibility when deploying to the server.
Add environment specific appsettings to configuration.
White space changes due to auto formatting on save.

### CODE REVIEW

File: IntelInventory/src/Configuration.cs [hunk 1]
```diff
@@ -26,13 +26,16 @@ private static IConfigurationRoot Initialize()
         {
             if (Root != null) return Root;
 
+            var dotNetEnvironment = Environment.GetEnvironmentVariable("DOTNET_ENVIRONMENT");
+            ;
```
Comment (eddieSkywalker90) [inline IntelInventory/src/Configuration.cs:30]: Extra unused semi-colon by itself

File: IntelInventory/appsettings.json [hunk 2]
```diff
@@ -1,87 +1,79 @@
...
+    "nsDefault": "http://intel.com/XML-Schema/TMG/Materials/MIV/WarehouseInventorysFileExchange",
+    "ns1": "http://intel.com/XML-Schema/TMG/Materials/MIV/FileExchange",
+    "ns2": "http://intel.com/XML-Schema/TMG/Materials/MIV/WarehouseInventorys",
+    "ns3": "http://www.w3.org/2001/XMLSchema-instance",
+    "nsSql": "urn:schemas-microsoft-com:xml-sql",
+    "SchemaLocation": "http://www.w3.org/2001/XMLSchema-instance",
+    "ExportedFileName": "Rinchem_WarehouseInventory_"
   },
 
   "Properties": {
```
Comment (eddieSkywalker90) [inline IntelInventory/appsettings.json:32]: Unless you see a need to keep the Properties section, I didn't actually end up using this entire section. . I just took the to and from locations that were used and put them under the FTP config section. Unless you see reason to keep Properties section we should remove.

**Review by eddieSkywalker90 — CHANGES_REQUESTED** [review]
Write 2 comments on appsettings.json + Configuration.cs. Neither are functional issues, just cosmetic.

File: IntelInventory/src/Configuration.cs (hunk 1 above)
Comment (Chem-Star) [reply IntelInventory/src/Configuration.cs:30]: Removed.

File: IntelInventory/appsettings.json (hunk 2 above)
Comment (Chem-Star) [reply IntelInventory/appsettings.json:32]: Agreed, removed.

**Pull request merged**